src/app/components/Form/TextField.tsx

```diff
@@ -0,0 +1,47 @@
+import Input from "./Input";
+
+const TextField = ({
+  id,
+  label,
+  placeholder,
+  type = "text",
+  required = false,
+  onChange,
+  onFocus,
+  onBlur,
+  value,
+  autoFocus = false,
+  autoComplete = "off",
+  disabled,
+  min,
+  max,
+}: React.InputHTMLAttributes<HTMLInputElement> & { label: string }) => (
+  <>
+    <label
+      htmlFor={id}
+      className="block font-medium text-gray-700 dark:text-white"
+    >
+      {label}
+    </label>
+    <div className="mt-1">
```

Comment:
Is this HTML structure generally usable for us or is this matching for the ones in this PR?

Reply:
Generally indeed. Makes our current code easier and future code :)